### ISSUE TITLE
Add a case contact count to the case follow-up view

### ISSUE THREAD

**Issue by kwa20** [hzi-braunschweig/SORMAS-Project#6456]
### Situation Description

Thus far, it is quite hard to follow-up cases and their contacts easily at the same time. Often enough, it is not explicitly clear if or how many contacts the case has and would require an intentional search. Therefore, as a user, I would like to easily see the cases contacts I am following up.
### Feature Description

- Add a column called 'Case contact count' to the case follow-up view next to symptom journal status
- Depict the count of the total number of case contacts
- Add filter icons (as seen for event groups) next to the number - upon clicking, open the contact follow-up view filtered by the cases contacts
### Additional Information

<img width="540" alt="caseContactCount" src="https://user-images.githubusercontent.com/70317594/130148511-92ad11f9-13e3-4b92-b51f-6e681742a1bb.PNG">

To figure out: 
- What if the case contacts under follow-up is less than the total number of case contacts? The filter might be confusing if it does not display the same numbers
    - Maybe display contacts under follow-up in parentheses after the total count?

**Comment by bernardsilenou** [comment]
@kwa20 
- This is a very nice nice feature to add, thanks
- With regard to your questions, I think such a situation can only occur if the followup duration of the case was extended.
- Since this view is the followup view, suggest we show only contacts of the case whose followup falls in the time interval. If users need to see all the contacts of each case, then then have to go to the detailed view.
- We can then add the option to compute number of contacts per case on detailed view + option to trigger the computation
- Would display total contacts in parenthesis not affect the sorting? if possible, it is good that we use one column for one variable only

**Comment by kwa20** [comment]
@bernardsilenou I think it would also be fine to just display the associated contacts under follow-up. Users just have to know that.
- we could either add this to the caption. Downside is that the caption becomes lengthy
- other option could be a hover info. Downside is that it's not directly visible

**Comment by bernardsilenou** [comment]
@kwa20 
- I also think the hover option is not a good idea
- I am not sure of what you mean by " associated contacts under follow-up", do you mean all the contacts of the case even if they are not under followup in the time specified?
- Another thing we can consider is when we migrate followup view to person based, if there are multiple diseases, then the total number of contacts per person may contain many people that the user do not need to followup on at the moment. 
- If the goal of the user is to know the sum of contacts per case/ case person, then we can count all contacts even the once that are not under followup at the moment. But if is to ` follow-up cases and their contacts easily at the same time`, then it may not be useful to show all contacts.
